Mathlib/Probability/Distributions/Gaussian/Real.lean

```diff
@@ -596,6 +597,15 @@ lemma gaussianReal_conv_gaussianReal {m₁ m₂ : ℝ} {v₁ v₂ : ℝ≥0} :
   simp only [Complex.ofReal_add, NNReal.coe_add]
   ring_nf
 
+/- The sum of two real Gaussian variables with means `m₁, m₂` and variances `v₁, v₂` is a real
+Gaussian distribution with mean `m₁ + m₂` and variance `v_1 + v_2`. -/
+lemma gaussianReal_add_indep_gaussianReal {Ω} {mΩ : MeasureSpace Ω} {P : Measure Ω}
```

Comment:
Two things:
- a suggestion for a name closer to our naming convention
- You have a `MeasureSpace` argument, which gives a default measure, but you also put a measure on it. I assume you meant `MeasurableSpace`?
```suggestion
lemma gaussianReal_add_gaussianReal_of_indepFun {Ω} {mΩ : MeasurableSpace Ω} {P : Measure Ω}
```